spec/acceptance/plugin_spec.rb

```diff
@@ -51,7 +51,8 @@ class {'jenkins':
       }
       EOS
 
-      apply2(pp)
+      apply(pp, catch_failures: true)
```

Comment:
Why shouldn't this use apply2 as well?

Reply:
Because ... i missed this one.
I force push a new code to fix this.